tests/integration/events/utils.go

```diff
@@ -79,14 +80,19 @@ func ExecuteRequestTestCase(
 
 	testRoutineClosedChan := make(chan struct{})
 	closeTestRoutineChan := make(chan struct{})
-	eventsChan, err := db.Events().Updates.Value().Subscribe()
+	eventsSub := db.Events().Subscribe(5, events.UpdateEventName)
```

Comment:
question: shouldn't this be unsubscribed-from at some point?

Reply:
node `Close` method takes care of unsubscribing.